tests/standalone/run_all.py

```diff
@@ -301,6 +304,24 @@ def checkRequirements(filename):
     return (True, "")
 
 
+def ParseCommandLineArguments(filename):
```

Comment:
Method names should follow snake_case.

Reply:
Also, where is this method called? I just want to understand its usage and try running it.

Reply:
Specifically, I am interested in understanding why this method returns nothing and what's exactly happening here. I don't understand it fully.

Reply:
> Also, where is this method called? I just want to understand its usage and try running it.

When I run the tests, the runner run_all.py automatically implements this method. That is what I observed when I used the runner on ClickUsing.py.

Reply:
> Specifically, I am interested in understanding why this method returns nothing and what's exactly happening here. I don't understand it fully.

parser.parse_args() will take the arguments provided when the program is run and interpret them according to the arguments that were added to the ArgumentParser object( in the code of the standalone test). So it assigns the CLI arg values to their corresponding variables and these assignments will exist in the global namespace and can be picked up by the standalone test. 
So it automatically returns these assignments to the global namespace. Hence there was no need to manually return anything.

Reply:
> Method names should follow snake_case.

okay. I'll implement that in the next commit. Thank you for the correction.